packages/plugins/data-collect/image-classification-data-collect/package.json

```diff
@@ -21,7 +21,8 @@
     "fs-extra": "^8.1.0",
     "glob": "^7.1.6",
     "glob-promise": "^3.4.0",
-    "uuid": "^7.0.2"
+    "uuid": "^7.0.2",
+    "@pipcook/pipcook-utils": "^0.5.0"
```

Comment:
IMO the plugin won't depend on core.

Reply:
What's the reason. Core provides some definitions of types of plugins

Reply:
IMO, the core will be our internal lib, and we need to keep fucking simple for plugin developers.